magni_bringup/scripts/pckg_version.py

```diff
@@ -5,9 +5,19 @@
 
 rospack = rospkg.RosPack()
 pkgs = ["move_basic", 
-	"fiducials",
 	"ubiquity_motor",
+	"fiducials",
 	"magni_robot",
```

Comment:
There seem to be a problem with fiducials and magni_robot packages, because rospack.get_path() cannot seem to find it. But all of there's subpackages can be found normally. Any thoughts?

Reply:
https://github.com/ros-infrastructure/rospkg/issues/18 seems like this is a known thing about rosversion. I think for now we can just pick one of the packages in the metapackage and get it's version number. Everything in a metapackage has the same version.

Reply:
Okay I wil choose a random one.